lib/config/configtest/resolve_config_parser_test.go

```diff
@@ -0,0 +1,85 @@
+package configtest
```

Comment:
this file should be with the resolve_config_parser.go file. You can name the package `config_test` and go will allow that

Reply:
You mean to keep the file at the same level as resolve_config_parser.go? I wanted to keep the tests and related data separate.

Reply:
thats correct. the test file and actual code go at the same level. go will let you append `_test` to the package name for the test file

Reply:
The latest commit has the changes as suggested.